astrokat/__main__.py

```diff
@@ -16,17 +15,19 @@ def session_options(parser,
                     x_short_opts=[],
                     x_long_opts=[]):
```

Comment:
The meaning of the `x` prefix isn't clear to me.  After looking through the code it seems to mean "options that must be removed".  Suggest the names are made a little more obvious.  E.g.
`hidden_short_opts` and `hidden_long_opts`,
or
`short_opts_to_remove` and `long_opts_to_remove`.

Reply:
updated to short_opts_to_remove and long_opts_to_remove